raster-test/src/test/scala/geotrellis/raster/GridBoundsSpec.scala

```diff
@@ -106,7 +105,7 @@ class GridBoundsSpec extends FunSpec with Matchers{
     it("should match the output of coordsIter") {
       val gbs = GridBounds(0, 0, 10, 10)
 
-      gbs.coordsIter.toSeq shouldBe gbs.coords.toSeq
+      gbs.coordsIter.toSeq shouldBe gbs.coordsIter.toSeq
```

Comment:
This breaks the test, we should keep the way it was before until the deprecated method is gone